actions/gerald-pr/action.yml

```diff
@@ -11,7 +11,7 @@ runs:
   using: "composite"
   steps:
       - name: Get All Changed Files
-        uses: Khan/actions@get-changed-files-v1
+        uses: ./actions/get-changed-files
```

Comment:
**note:** This seems like a more appropriate way to reference this action. With any given release we would want to use the version of the action that was included in that release.